src/dash-table/dash/DataTable.js

```diff
@@ -256,6 +253,26 @@ export const propTypes = {
             ])
         }),
 
+        /**
+         * There are two `sort_as_none` flags in the table.
+         * This is the  column-level editable flag and there is
+         * also the table-level `sort_as_none` flag.
+         *
+         * These flags determine how the content of the table is
+         * sorted.
+         *
+         * If the column-level `sort_as_none` flag is set it overrides
+         * the table-level `sort_as_none` flag for that column.
+         *
+         * An array of string, number and boolean values that are treated as `None`
+         * when sorting is applied to the column.
+         */
+        sort_as_none: PropTypes.arrayOf(PropTypes.oneOfType([
```

Comment:
Actually, would we be better off calling this `sort_as_null`? That seems more language-agnostic, and we do use `null` elsewhere in the names (like `validation.allow_null`). Am I right to assume that the other nully values mentioned below (`undefined`, `NaN`) automatically sort as `null`?

Also in the docstring - it's not a flag anymore.

Reply:
Sure, `_null` it is.